_posts/2024-03-27-website-relaunch.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Website Relaunch"
+author: straight-shoota
+summary: "Fresh design and new content on crystal-lang.org"
+image: /assets/blog/2024/2024-03-27-website-homepage-new.webp
+categories: project
+tags: [website]
+---
+
+The Crystal website here at [crystal-lang.org](https://crystal-lang.org) has received a major overhaul.
+
+This project has been in the making for a long time, with the first design previews dating back at least to 2020. Many people have been involved in the process which made very slow progress because there were always more important things to do (such as developing a programming language).
+But the limitations of the old website have been quite substantial and finally we managed to complete the relaunch.
+
+Some of the highlight improvements:
+
+* Responsive layout that fits small and large viewports
+* Refined typography
+* Enhanced site navigation system
+* completely reorganised [homepage](/)
+* completely reorganised [_Install_ page](/install/)
+* completely reorganised [_Community_ page](/community/)
+* new [_Releases_ page](/releases/)
```

Comment:
Same Casing in each sentence. As a note, I prefer the US "reorganized", but this is entirely up to you.

```suggestion
* Completely reorganised [homepage](/)
* Completely reorganised [_Install_ page](/install/)
* Completely reorganised [_Community_ page](/community/)
* New [_Releases_ page](/releases/)
```

Reply:
Actually, I do as well. Spellchecker seems to be set to British English 🤷